### PR TITLE
Scan configuration and workflow changes to allow Whitesource scan for Dashboards and Opensearch plugins

### PR DESCRIPTION
**Pull request by sreekarjami**
### Description
Scan configuration and workflow changes to allow Whitesource scan for Dashboards and Opensearch plugins
 
### Issues Resolved
[List any issues this PR will resolve]

###Tests:
Works locally 
 
### Check List
- [x] Commits are signed per the DCO using --signoff 

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.
For more information on following Developer Certificate of Origin and signing off your commits, please check [here](https://github.com/opensearch-project/OpenSearch/blob/main/CONTRIBUTING.md#developer-certificate-of-origin).

### CODE REVIEW

File: .github/workflows/check-vulnerability-whitesource.yml [hunk 1]
```diff
@@ -44,36 +44,28 @@ jobs:
...
+          sudo npm install --global yarn@1.22.10
           export PATH=$PATH:/opt/gradle/gradle-6.7/bin
-          gradle -v; mvn -v ; npm -v; yarn -v
+          gradle -v; mvn -v; node -v; npm -v; yarn -v
           # This step is needed to avoid build failures in few plugins
           # No ETA on when this dependency can be removed 
-          git clone https://github.com/opensearch-project/OpenSearch.git
-          cd OpenSearch
-          gradle publishToMavenLocal; cd ..
+          git clone -b 1.0 https://github.com/opensearch-project/OpenSearch.git
```
Comment (peterzhuamazon) [inline .github/workflows/check-vulnerability-whitesource.yml:65]: Please do not hardcode to 1.0 branch.

File: .github/workflows/check-vulnerability-whitesource.yml [hunk 2]
```diff
@@ -44,36 +44,28 @@ jobs:
...
-          aws-secret-access-key: ${{ secrets.AWS_MAVEN_SECRET_ACCESS_KEY }}
-          aws-region: us-west-2
       - name: Vulnerability Scan
         id: vulnerability_scan
         env:
           wss_apikey: ${{ secrets.WSS_API_KEY }}
         run: |
           export PATH=$JAVA_HOME:$PATH
-          cd standalone-tools/vulnerability-scan
+          cd  tools/vulnerability-scan
```
Comment (peterzhuamazon) [inline .github/workflows/check-vulnerability-whitesource.yml:53]: Remove extra space.

File: .github/workflows/check-vulnerability-whitesource.yml [hunk 3]
```diff
@@ -44,36 +44,28 @@ jobs:
...
           sudo mkdir /opt/gradle
           sudo unzip -d /opt/gradle gradle-6.7-bin.zip
-          wget -qO - https://rpm.nodesource.com/setup_14.x | sudo bash -
-          sudo yum install -y nodejs
-          wget -qO - https://dl.yarnpkg.com/rpm/yarn.repo | sudo tee /etc/yum.repos.d/yarn.repo
-          sudo rpm --import https://dl.yarnpkg.com/rpm/pubkey.gpg
-          sudo yum install yarn -y
+          wget -qO - https://rpm.nodesource.com/setup_10.x | sudo bash -
+          sudo yum install -y nodejs-10.23.1
+          sudo npm install --global yarn@1.22.10
```
Comment (peterzhuamazon) [inline .github/workflows/check-vulnerability-whitesource.yml:60]: Please do not hard code nodejs and yarn version.

File: tools/vulnerability-scan/wss-scan.config [hunk 4]
```diff
@@ -1,3 +1,3 @@
 baseDirPath=$(pwd)
 gitBasePath=https://github.com/opensearch-project/
-gitRepos=alerting,alerting-dashboards-plugin,anomaly-detection,anomaly-detection-dashboards-plugin,asynchronous-search,common-utils,dashboards-notebooks,dashboards-reports,dashboards-visualizations,data-prepper,index-management,index-management-dashboards-plugin,job-scheduler,k-NN,opensearch-cli,performance-analyzer,performance-analyzer-rca,perftop,security,security-dashboards-plugin,sql,trace-analytics
+gitRepos=OpenSearch,alerting,alerting-dashboards-plugin,anomaly-detection,anomaly-detection-dashboards-plugin,asynchronous-search,common-utils,dashboards-notebooks,dashboards-reports,dashboards-visualizations,data-prepper,index-management,index-management-dashboards-plugin,job-scheduler,k-NN,opensearch-cli,performance-analyzer,performance-analyzer-rca,perftop,security,security-dashboards-plugin,sql,trace-analytics
```
Comment (peterzhuamazon) [inline tools/vulnerability-scan/wss-scan.config:3]: I thought this is for both OpenSearch and Dashboards why is there only OpenSearch addition?

File: .github/workflows/check-vulnerability-whitesource.yml (hunk 1 above)
Comment (sreekarjami) [reply .github/workflows/check-vulnerability-whitesource.yml:65]: This is temporarily needed, until we have Maven central.

File: .github/workflows/check-vulnerability-whitesource.yml (hunk 3 above)
Comment (sreekarjami) [reply .github/workflows/check-vulnerability-whitesource.yml:60]: Dashboards plugins are throwing build errors, so had to match the node and yarn version mentioned in Dashboards.

File: tools/vulnerability-scan/wss-scan.config (hunk 4 above)
Comment (sreekarjami) [reply tools/vulnerability-scan/wss-scan.config]: Change made.

**Comment by camerski** [comment]
Replaced by https://github.com/opensearch-project/opensearch-build/pull/77

**Pull request closed without merging**